src/dragonball/src/dbs_virtio_devices/src/lib.rs

```diff
@@ -201,15 +204,20 @@ pub enum Error {
     #[error("virtio-vsock error: {0}")]
     VirtioVsockError(#[from] self::vsock::VsockError),
 
-    #[cfg(feature = "virtio-net")]
-    #[error("Virtio-net error: {0}")]
-    VirtioNetError(#[from] crate::net::NetError),
-
     #[cfg(feature = "virtio-fs")]
     /// Error from Virtio fs.
     #[error("virtio-fs error: {0}")]
     VirtioFs(fs::Error),
 
+    #[cfg(feature = "vhost")]
+    #[error("vhost error: {0}")]
+    /// Error from vhost-net.
+    VhostNet(#[from] vhost_rs::Error),
+
+    #[cfg(feature = "virtio-net")]
+    #[error("tap device operation error: {0}")]
+    TapDeviceError(#[source] TapError),
```

Comment:
Why remove VirtioNetError? I think we should distinguish errors from VirtioNet and VhostNet

Reply:
Fixed: Remove `TapError` from dbs-virtio-devices's Error, and add `VirtioNet` and `VhostNet` two fields.